src/components/SubmitClassificationForm.jsx

```diff
@@ -8,8 +8,7 @@ import PropTypes from 'prop-types';
 import { connect } from 'react-redux';
 import { config } from '../config';
 import {
-  createClassification, submitClassification,
-  setSubjectCompletionAnswers,
+  submitClassification, setSubjectCompletionAnswers,
```

Comment:
`submitClassification` was not being used

Reply:
Wait, you mean `createClassification` was not being used?

Reply:
Oh, my bad. Yes, `createClassification` is not being used by this component.